src/lib.rs

```diff
@@ -17,12 +17,46 @@ use std::collections::HashSet;
 use std::path::PathBuf;
 use std::process::Command;
 
+///
+/// Executes the LLVM host repository cloning for stage 1 MUSL builds.
+///
+pub fn clone_host() -> anyhow::Result<()> {
+    let destination_path = PathBuf::from(LLVMPath::DIRECTORY_LLVM_HOST_SOURCE);
+    if destination_path.exists() {
+        println!(
```

Comment:
As we don't use llvm.lock for the host, I believe it's good enough to just print a warning and skip if it's already cloned. But let me know if you have some other thoughts.

Reply:
Use `eprintln!` please so it writes to `stderr`.

Reply:
Done.